config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    get 'articles/index'
+  end
```

Comment:
This is an auto-generated block of code. As we say in Swedish: "kaka på kaka" (see code below)

Reply:
Removed